main/ui/src/main/resources/fxml/health_start.fxml

```diff
@@ -20,9 +16,10 @@
 		<Insets topRightBottomLeft="12"/>
 	</padding>
 	<children>
-		<Label text="TODO: Invalid vault config" visible="${controller.invalidConfig}" managed="${controller.invalidConfig}"/>
+		<Label text="%health.start.introduction" wrapText="true"/>
 
-		<Label text="TODO: Blabla going to health check now...:"/>
+		<Label text="%health.start.configInvalid" visible="${controller.invalidConfig}" managed="${controller.invalidConfig}"/>
```

Comment:
Should contain `wrapText="true"` as well, I guess. (Keeping l10n in mind)

Reply:
fixed in eb19616